spring-kafka/src/main/java/org/springframework/kafka/support/serializer/JsonDeserializer.java

```diff
@@ -43,7 +43,7 @@
  */
 public class JsonDeserializer<T> implements Deserializer<T> {
 
-	protected final Logger logger = LoggerFactory.getLogger(getClass());
+	protected final Log logger = LogFactory.getLog(getClass());
```

Comment:
Well, there is no any usage of this property in this class any more.
So, it would be better to remove it altogether like I did in the `JsonSerializer`.
That was just my omission to leave it here.

Thank you for catching that anyway!